third_party/move/move-compiler/src/parser/syntax.rs

```diff
@@ -1522,7 +1524,7 @@ fn parse_for_loop(context: &mut Context) -> Result<(Exp, bool), Box<Diagnostic>>
 }
 
 // Match = "match" "(" <Exp> ")" "{" ( <MatchArm> ","? )* "}"
-// MatchArm = <Bind> ( "if" <Exp> )? "=>" <Exp>
+// MatchArm = <BindList> ( "if" <Exp> )? "=>" <Exp>
```

Comment:
I note that `BindList` is documented around line 819 and says it can be "`zero, one, or more bindings`"?  Is zero actually possible for a `BindList`?  If not, can you correct that comment.  If so, what happens here with 0 bindings?

Reply:
I think `parse_comma_list` accepts `()`
<img width="448" alt="Screenshot 2024-08-08 at 5 50 47 PM" src="https://github.com/user-attachments/assets/2d42d8cf-7b19-4c3d-9b2b-5e06cc064bac">